### PR TITLE
Get requests

### PR DESCRIPTION
**Pull request by Mirrorlock**
Get all users and retrieve avatar lambda functions ready

### CODE REVIEW

File: get/getAllItems/index.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var AWS = require('aws-sdk');
+var Joi = require('joi');
+
+AWS.config.update({region: 'us-east-2'});
+
+exports.handler = async (event) => {    
+    var ddb = new AWS.DynamoDB();
```
Comment (Vpvladimirov) [inline get/getAllItems/index.js]: Use let/const instead of var

File: get/getAllItems/index.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+exports.handler = async (event) => {    
+    var ddb = new AWS.DynamoDB();    
+    var params = {
+        "TableName" : "Users"
+    };
+    console.log("Fetching all users from db...");
+    var result;
+    
+    try{
```
Comment (Vpvladimirov) [inline get/getAllItems/index.js]: Add spaces before opening and after closing brackets { }

File: get/getAllItems/index.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+var AWS = require('aws-sdk');
+var Joi = require('joi');
```
Comment (Vpvladimirov) [inline get/getAllItems/index.js]: You don't need joi when retrieving the Users but only when inserting new ones.

File: get/getAllItems/package.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "getAllItems",
+    "version": "1.0.0",
+    "description": "A node project to get all users from db",
+    "main": "index.js",
+    "scripts": {
+        "start": "npm run dev"
+    },
+    "dependencies": {
+        "joi": "^14.3.1"
```
Comment (Vpvladimirov) [inline get/getAllItems/package.json]: You don't need joi when retrieving the units but only when inserting new ones.

File: get/getAllItems/index.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    var params = {
+        "TableName" : "Users"
+    };
+    console.log("Fetching all users from db...");
+    var result;
+    
+    try{
+        result = await ddb.scan(params).promise();
+        result = result.Items
+
```
Comment (Vpvladimirov) [inline get/getAllItems/index.js]: Is it possible to be changed to:
```suggestion
(await ddb.scan(params).promise()).Items
```

**Review by Vpvladimirov — CHANGES_REQUESTED** [review]

File: get/getAllItems/index.js (hunk 5 above)
Comment (Mirrorlock) [reply get/getAllItems/index.js]: I agree, i will change it

**Pull request merged**